### ISSUE TITLE
Uploading releases to central repository

### ISSUE THREAD

**Issue by ssalonen** [Scrin/ruuvitag-common-java#3]
Hi

It would be great to have the releases available on central maven repository, to make it easier to use in downstream projects.

See:
https://maven.apache.org/repository/guide-central-repository-upload.html

**Comment by Scrin** [comment]
This is actually on my todo-list, but no estimate yet as the initial setup for that is a bit more complicated process. Figured the "fake repository on github" approach is better than nothing in the meantime

**Comment by ssalonen** [comment]
I suppose the maven central upload is the same status as before?

At minimum, do you mind creating manual release tag here for the 1.1.0?

This way I can have this manually built jar published to openHAB maven repositories (for https://github.com/openhab/openhab-addons/pull/19638 )

**Comment by Scrin** [comment]
The status currently is the same in the sense that I've not done it, yes. I've also created the v1.1.0 release, I hope I got it right since it has been years since releasing the previous version and I've lost my notes on the matter

However what's different is that this is no longer on my todo-list, as I no longer actively maintain any Java projects and I don't use Java in my current dayjob either, so I lack the proper development environments and my Java knowledge is likewise getting outdated. If you want to take over maintaining this project and publish to the central repository, I think that can be arranged. Otherwise this project stays here on a best-effort basis, mainly relying on other contributors

**Comment by ssalonen** [comment]
Yes this is all good! It would be pretty same situation for me, does not matter too much

Thanks for the release!
